bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/BRuntime.java

```diff
@@ -275,66 +179,4 @@ public void notifyFailure(ErrorValue error) {
     }
 
 
-    private void scheduleNextFunction(FPValue<?, ?> func, Strand strand, String strandName,
```

Comment:
Shall we refactor the getSyncMethodInvokeResult method as well.

Reply:
By refactor do you mean that I should remove the function? Or to simplify code by reusing? This function is already used in Kafka